src/features/timeline/TimelineIndexPage.js

```diff
@@ -44,22 +42,30 @@ const CycleIndexPage = ({ entryId }) => {
     })
   }, [selectedDate])
 
+  const notHasPlacedPeriod =
+    !entries.length || !entries.filter((s) => s.tags.length).length
+
+  useEffect(() => {
+    if (notHasPlacedPeriod) {
+      navigate("/incomplete")
+    }
+  }, [notHasPlacedPeriod])
+
   return (
     <AppLayout>
-      <AppMainToolbar>
-        <DatePicker date={selectedDate} />
-        <IconButton
-          aria-label="Scroll to today"
-          onClick={(event) => {
-            navigate(`/timeline`)
-          }}
-          style={{ marginLeft: "auto" }}
-        >
-          <Today />
-        </IconButton>
-      </AppMainToolbar>
-
       <AppPage>
+        <AppMainToolbar>
```

Comment:
**suggestion:** Usually use the `AppMainToolbar` outside of the `AppPage`

Reply:
Moved into the AppLayout